src/cohortMembers/cohortMembers.controller.ts

```
@@ -451,6 +453,16 @@ export class CohortMembersController {
         );
       }
 
+      if (body?.cohortId && !isUUID(body.cohortId)) {
+        return APIResponse.error(
+          res,
+          apiId,
+          API_RESPONSES.BAD_REQUEST,
+          'Invalid cohortId format. Must be a valid UUID.',
```

ESLint

[error] Replace `'Invalid·cohortId·format.·Must·be·a·valid·UUID.'` with `"Invalid·cohortId·format.·Must·be·a·valid·UUID."`

(prettier/prettier)